### ISSUE TITLE
MsAdalAngular6Service.isAuthenticated should be boolean instead of string

### ISSUE THREAD

**Issue by sbley** [manishrasrani/ms-adal-angular6#1]
These are the type definitions of ms-adal-angular6.service.d.ts:
```
/// <reference path="../../../node_modules/@types/adal/index.d.ts" />
import { Observable } from 'rxjs';
export declare class MsAdalAngular6Service {
    private adalConfig;
    private context;
    constructor(adalConfig: any);
    readonly LoggedInUserEmail: string;
    readonly LoggedInUserName: any;
    login(): void;
    logout(): void;
    GetResourceForEndpoint(url: string): string;
    RenewToken(url: string): void;
    acquireToken(url: string): Observable<any>;
    getToken(url: string): string;
    handleCallback(): void;
    readonly userInfo: adal.User;
    readonly accessToken: string;
    readonly isAuthenticated: string;
}
```
What bothers me is that `isAuthenticated` is of type `string`. It should be `boolean`.

**Comment by manishrasrani** [comment]
@sbley this is fixed in commit - https://github.com/manishrasrani/ms-adal-angular6/commit/972bc33ed6a0da88b75009da0cf1850fa2f6df1b

**Comment by mchint01** [comment]
Is this commit available on npm? We don't see it

**Comment by manishrasrani** [comment]
@mchint01 - Sorry missed publishing this. Its now available on npm microsoft-adal-angular6@1.1.0
